framework/src/postprocessors/ElementW1pError.C

```diff
@@ -34,7 +34,8 @@ ElementW1pError::ElementW1pError(const InputParameters & parameters) :
 Real
 ElementW1pError::getValue()
 {
-  return std::pow(ElementIntegralPostprocessor::getValue(), 1./_p);
```

Comment:
Right... Then the loss of elegance is outweighing the convenience of a scaling parameter. I'd vote against this change (if I had a vote :-D). Just use a second PP to scale the value.